js/runner.go

```diff
@@ -269,6 +272,25 @@ func (r *Runner) SetOptions(opts lib.Options) {
 	}
 }
 
+// InitConsole initialises a console based on the runner config.
+func (r *Runner) InitConsole() error {
```

Comment:
I thinks This should be just part of SetOptions above. Also maybe set the console by default to `NewConsole` and then just override it if necessary.

Reply:
Hmm. I could move it. But then we would be initialising a console inside a function named `SetOptions` which doesn't sound right since a function should not implicitly do more than what it advertises via the function name. But I see that the `Limiter` (ratelimiter) is initialised inside `SetOptions`, so I guess it is okay although not ideal.